stock_picking_manual_package/wizards/stock_picking_manual_package_wizard_view.xml

```diff
@@ -11,6 +11,13 @@
                 </group>
                 <p>Set how many lines you want to 'Put in pack'</p>
                 <group>
+                    <field name="picking_product_ids" invisible="1" />
```

Comment:
I'd suggest you to enhance the <p> message in order to help easily user to deal with product_ids field (without hover).